GoogleMulticastAppDelegate.podspec

```diff
@@ -0,0 +1,72 @@
+Pod::Spec.new do |s|
+  # TODO: Is `GoogleMulticastAppDelegate` name fine?
+  s.name             = 'GoogleMulticastAppDelegate'
+  s.version          = '7.5.0'
```

Comment:
Open question for discussion. Should the versioning match GoogleUtilities? For SPM, should this be part of the GoogleUtilities package?

Reply:
Weighing in: I think the versioning should match GoogleUtilities. Mainly because I think having two GoogleUtilities-related versions to track could be error-prone.